client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -424,19 +426,39 @@ const PluginMeta = React.createClass( {
 		analytics.ga.recordEvent( 'Plugins', 'Clicked Update All Sites Plugin', 'Plugin Name', this.props.pluginSlug );
 	},
 
+	getExtensionPath( plugin ) {
+		const pluginSlug = get( plugin, 'slug' );
+		const sections = sectionsModule.get();
+		const index = findIndex( sections, { name: pluginSlug } );
```

Comment:
What about using lodash's `find` instead? That'd give us the complete object found at `[ index ]` (or `undefined` if it's not found). I think that could simplify the code a bit...